data size paragraph

dvnagorny opened the pull request.

nagorny committed:
data size paragraph

anton-malakhov commented on papers/oleksandr_pavlyk/sklearn/sklearn.rst:
```diff
@@ -14,6 +14,12 @@ Specifically, daal4sklearn optimizes Principal Component Analysis (PCA), Linear
 
+Originally scikit-learn uses multiprocessing approach to parallelize computation. 
+Unfortunately as the result it can have significant memory pollution as all cloned processes can use their own copy of all data.
+It leads to impossibility to effectivly utilize many-cores architectures as Intel |R| Xeon Phi |TM| for big workloads.
```
@oleksandr-pavlyk We need to reword it for smoother reading

anton-malakhov commented on papers/oleksandr_pavlyk/sklearn/sklearn.rst:
```diff
@@ -23,45 +29,10 @@ Daal4sklearn is enabled by default and provides a simple API to toggle these opt
         dispatcher.enable()
 
 We prepared several benchmarks to demonstrate performance that can be achieved with Intel |R| DAAL.
+There [sklearn_benches]_ you can see these benchmarks. There you can see small extract from these codes.
```
this needs rewording as well

anton-malakhov reviewed (dismissed):
While it's ok to put these thoughts into the paper, it needs rewording, @oleksandr-pavlyk could you please handle that for Denis?

oleksandr-pavlyk committed:
Rewording per Anton's feedback. (#24)

Changed uses of (p, n) in the benchmarking code to uses of (rows, cols)
to be consistent with names used in the table
